static/app/utils/replays/getFrameDetails.tsx

```diff
@@ -224,6 +224,20 @@ const MAPPER_FOR_FRAME: Record<string, (frame) => Details> = {
     title: 'User Focus',
     icon: <IconUser size="xs" />,
   }),
+  'app.foreground': () => ({
+    color: 'blue300',
+    description: 'Video started recording',
```

Comment:
While this is true, I would describe this in terms of what the application is doing, and not what we/SDK is doing.

Reply:
do you have any ideas of what to put instead?

Reply:
@billyvg do you have strong opposing thoughts here? i'm also ok with "application went into the background" and variations

Reply:
I'm ok with "Replay <status>" - I suppose I want to avoid "video" because of web replays and how we want to make sure that people know that replays aren't videos

Reply:
okay let's go with the replay terminology!